### PR TITLE
Check for existing issues in Jira

### LINKED ISSUE #54: Don't spam Jira with the same issue
Instead of duplicate issues, just annotate existing issues if they exist in Jira. It doesn't make sense to create the same issue over and over.

### PR DESCRIPTION
**Pull request by epaulet**
fixes #54

### CODE REVIEW

File: lib/integrations/jira.rb [hunk 1]
```diff
@@ -9,26 +9,62 @@ def self.key
...
+        create_issue
+      end
+    end
+
+    private
+
+    def update_issue(issue_id)
+      params = {
+        body: {
+          update: { labels: [{ add: 'RepeatedFailures' }] },
```
Comment (shosti) [inline lib/integrations/jira.rb]: Don't know what we do already, but it seems like things like labels and priorities should be customizable probably (they vary a lot between installations I think). I know this is v1, but can you maybe rewrite in a way that customization can be added later without a lot of pain?

File: lib/integrations/jira.rb [hunk 2]
```diff
@@ -75,5 +111,12 @@ def jira_base_url
       base_url = settings[:jira_base_url]
       base_url.last == "/" ? base_url.chop : base_url
     end
+
+    def headers
+      {
+        basic_auth: { username: settings[:username], password: settings[:password] },
```
Comment (shosti) [inline lib/integrations/jira.rb]: lol we *really* need to fix this

Comment (epaulet) [inline lib/integrations/jira.rb]: Yeah that's it's own issue. I may need some help tomorrow with this btw :)

**Comment by shosti** [comment]
LGTM as v1, but I think we still have a long way to go before JIRA integration is actually useful...

**Comment by shosti** [comment]
:ship:

**Pull request merged**